### PR TITLE
[DATA-277] Update date/time format to match datamanager

### PR DESCRIPTION
**Pull request by JohnN193**
SLAM PR for this https://github.com/viamrobotics/slam/pull/73

updates to the slam repo to use the same timeformat used by other services. Will not merge until integration tests pass, which requires updates from the slam repo first

https://viam.atlassian.net/browse/DATA-277

### CODE REVIEW

File: services/slam/builtin/builtin.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ const (
 	parsePortMaxTimeoutSec      = 60
 	// TODO change time format to .Format(time.RFC3339Nano) https://viam.atlassian.net/browse/DATA-277
 	// time format for the slam service.
```
Comment (tessavitabile) [inline services/slam/builtin/builtin.go:59]: [nit] Remove TODO. There are also a few other TODOs for DATA-277, so please make sure to remove them all.

File: services/slam/builtin/builtin.go [hunk 2]
```diff
@@ -57,7 +57,7 @@ const (
...
-	slamTimeFormat        = "2006-01-02T15_04_05.0000"
+	slamTimeFormat        = time.RFC3339Nano
```
Comment (kkufieta) [inline services/slam/builtin/builtin.go:60]: Remove the comment

**Review by kkufieta — APPROVED** [review]
LGTM - just a comment about removing the comment

**Review by tessavitabile — APPROVED** [review]
LGTM mod nit

**Pull request merged**